changelog.d/219.bugfix

```diff
@@ -0,0 +1 @@
+fix towncrier failure in Github Actions
```

Comment:
A couple of things here. 

Firstly: this isn't fixing a bug that a user would see - it's more of a development thing, so it would be better to call this `219.misc`, so that it ends up in the "Internal changes" section of the changelog.

But in this case, since we haven't made a release since #210, it might be more appropriate to give it the same changelog as #210. (Just create a `219.misc` with exactly the same contents as `210.misc`). Then, when we come to do the release, towncrier will merge the two entries together and link to both PRs.

Reply:
Cool thanks for reviewing! I made the requested changes and updated the PR.